sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetFileFormat.scala

```diff
@@ -307,6 +308,10 @@ class ParquetFileFormat
     hadoopConf.set(
       ParquetWriteSupport.SPARK_ROW_SCHEMA,
       requiredSchema.json)
+    hadoopConf.set(
+      SQLConf.SESSION_LOCAL_TIMEZONE.key,
+      sparkSession.sessionState.conf.sessionLocalTimeZone
+    )
```

Comment:
nit: we can join this line to the previous line.

Reply:
Do you mean:
```
hadoopConf.set(
      SQLConf.SESSION_LOCAL_TIMEZONE.key, sparkSession.sessionState.conf.sessionLocalTimeZone)
```

? If so, there are several other `hadoopConf.set()` calls that could be similarly formatted. Would you like me to fix them as well?

Reply:
I think he meant simply moving `)` up ..

Reply:
```
hadoopConf.set(
  SQLConf.SESSION_LOCAL_TIMEZONE.key,
  sparkSession.sessionState.conf.sessionLocalTimeZone)
```

Reply:
> If so, there are several other hadoopConf.set() calls that could be similarly formatted. Would you like me to fix them as well?

I wouldn't change styles if it's in other places to make this easier to manage (e.g., backport, revert).